fapolicy_analyzer/ui/strings.py

```diff
@@ -118,3 +118,24 @@
 GET_GROUPS_LOG_ERROR_MSG = _(
     "An error occurred trying to retrieve the group list. Please try again."
 )
+
+TRUST_DB_READ_FAILURE_DIALOG_TITLE = _("Trust Database Read Failure")
+TRUST_DB_READ_FAILURE_DIALOG_TEXT = _(
```

Comment:
Is center justifying the text in the string itself a common practice? Seems like the UI component should be responsible for that.

Reply:
I'm sure there is support in the Gtk ecosystem to have the widget format the text string. I used the same simplistic stuff an hbox w/a text string implementation as I used for the interrupted edit session detected dialog.  I can pursue this if you want; your call.

Reply:
Fwiw, I wanted the possible causes numbering to be left aligned.

Reply:
Added a call to center the label's text. Done.